src/output.rs

```diff
@@ -150,6 +153,8 @@ impl OutputTypeDetection for TxOut {
                 return OutputType::OpReturn(OpReturnFlavor::Len20Byte);
             } else if self.is_opreturn_with_len(80) {
                 return OutputType::OpReturn(OpReturnFlavor::Len80Byte);
+            } else if self.is_opreturn_bip47_payment_code() {
+                return OutputType::OpReturn(OpReturnFlavor::Bip47PaymentCode);
```

Comment:
I think this won't ever be reached as we check `is_opreturn_with_len(80)` beforehand. I can fix it up.

Reply:
fixed in 903d72b